### PR TITLE
fixes #11337 - adds Package Group details to UI

### PR DESCRIPTION
**Pull request by cfouant**

### CODE REVIEW

**Comment by johnpmitsch** [comment]
Looks good as far as I can tell.

Two ideas that we had thrown out there:
1) include package groups link in content dropdown menu
2) have a way of seeing the packages for each package group

Do you see any concrete reason to add either/both of these?
@jlsherrill and @ehelms what do you think?

**Comment by cfouant** [comment]
@johnpmitsch - after discussing this with @ehelms and @jlsherrill we decided to keep them as simple tables accessible much like it is currently visible in content view versions, and off the top menu.

**Comment by johnpmitsch** [comment]
@cfouant gotcha :+1:

File: engines/bastion_katello/app/assets/javascripts/bastion_katello/repositories/details/views/repository-manage-package-groups.html [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        <th bst-table-column><span translate>Name</span></th>
+      </thead>
+      <tbody>
+        <tr bst-table-row ng-repeat="packageGroup in detailsTable.rows">
+          <td bst-table-cell>{{ packageGroup.name }}</td>
+        </tr>
+      </tbody>
+    </table>
+  </div>
+</div>
```
Comment (jlsherrill) [inline engines/bastion_katello/app/assets/javascripts/bastion_katello/repositories/details/views/repository-manage-package-groups.html]: I currently see it say '0 Selected' in the upper right corner.  I think you can fix this by adding an empty "selection-summary" data-block div.

**Comment by cfouant** [comment]
@jlsherrill good catch - i've updated. Thanks!

**Comment by jlsherrill** [comment]
ACK

**Pull request merged**